include/samurai/level_cell_array.hpp

```diff
@@ -72,9 +78,14 @@ namespace samurai
         using cell_t              = Cell<dim, interval_t>;
         using index_t             = typename interval_t::index_t;
         using value_t             = typename interval_t::value_t;
+        using coord_index_t       = typename interval_t::coord_index_t;
```

Comment:
`value_t` and `coord_index_t` are the same type. The latter will be removed in a next PR.